tools/meteor-services/auth.js

```diff
@@ -65,6 +65,9 @@ var loggedInAccountsConnection = function (token) {
     connection.close();
 
     if (err.error === 403) {
+      // Better than saying nothing at all
+      Console.error("warning: login token was rejected by the server");
```

Comment:
Also, I'd question having the word `warning:` in a `Console.error`.  Since it seems like this was an error actually preventing you from continuing, it seems that removing `warning:` would be more appropriate.